run_tests.py

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env python3
+import argparse
+import logging
 import unittest
 
-if __name__ == '__main__':
+logging.basicConfig(level=logging.DEBUG)
+__log__ = logging.getLogger(__name__)
```

Comment:
Is `__log__` a special name?

Reply:
Not really, it's just visually more distinctive and clear that it's not part of business logic. There is no standard, everyone uses own way, `log.`, `_log.`, `log_.`...

Reply:
https://docs.python.org/3/reference/lexical_analysis.html#reserved-classes-of-identifiers
>`__*__`: System-defined names. These names are defined by the interpreter and its implementation (including the standard library).

I wouldn't recommend making arbitrary new `__names__`.

Reply:
Good point, thanks. Fixed.